config/routes.rb

```diff
@@ -1,4 +1,14 @@
 Rails.application.routes.draw do
+  resource  :session
+  resources :passwords, param: :token
+
+  namespace :registration do
```

Comment:
Registration is a resource, it is not a namespace. You can draw this route like:
`resources :registrations, only: %i[new create]`